examples/src/agents/sql.ts

```diff
@@ -28,6 +34,4 @@ export const run = async () => {
       2
     )}`
   );
-
-  db.close();
```

Comment:
Is there no equivalent to this in the typeorm data source thing or something?

Reply:
ah you're right, should be using ..destroy here. Will update